styles.css

```diff
@@ -112,3 +112,98 @@ header {
     display: block;
 }
 
+.submit-button {
+    margin-left: 68%;
+    margin-top: 60%;
+    padding: 10px;
+    font-size: 20px;
+    background-color: #B3ECFF;
+    color: #000;
+    border-radius: 20%;
+    transition: box-shadow 0.5s ease-in-out, background-color 0.5s ease-in-out;
+    box-shadow:  2px 5px rgba(0, 0, 0, 0.25);
+    display: none;
+}
+
+.submit-button:hover {
+    background-color: #B3FFD9;
+}
+
+.show-button {
+    display: block;
+}
+
+.modal {
+    position: absolute;
+    display: flex;
+    width: 100vw;
+    height: 100vh;
+    background: #5433FF;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #A5FECB, #20BDFF, #5433FF);  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #A5FECB, #20BDFF, #5433FF); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+    justify-content: center;
+    align-items: center;
+}
+
+.modal-content {
+    width: 700px;
+    height: 200px;
+    background-color: white;
+    padding: 20px;
+}
+
+.modal-title {
+    height: 30%;
+    width: 96%;
+    margin: auto;
+    text-align: center;
+    font-size: 35px;
+    background-image: linear-gradient(to top, #FC00FF, #00DBDE); 
```

Comment:
I think that it could be `background: linear-gradient....`
ANd then, I don't think next line is needed

Reply:
It has to be there, bc if there's not it shows only linear background, not text.